### PR TITLE
docker: `prophet` is no longer a part of the image

### PR DESCRIPTION
**Pull request by abitrolly**
Removed from default `requirements.txt` here https://github.com/mindsdb/lightwood/pull/897/files

### CODE REVIEW

**Comment by abitrolly** [comment]
It also ships with wheels https://github.com/mindsdb/lightwood/pull/911

**Comment by ZoranPandovski** [comment]
Thanks, we will merge this after Lightwood merge and release.

**Comment by abitrolly** [comment]
@ZoranPandovski the commit that removes `prophet` from defaults was already merged and released.

**Comment by paxcema** [comment]
As of today, the fix @abitrolly mentions has been merged into Lightwood staging, with a release scheduled for next week.

**Comment by abitrolly** [comment]
@paxcema I thought that `prophet` was removed from `lightwood` defaults in June https://github.com/mindsdb/lightwood/pull/897/files and released also that month.

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**
